lib/manageiq/loggers/journald.rb

```diff
@@ -28,12 +44,59 @@ def add(severity, message = nil, progname = nil)
         end
 
         message = formatter.call(format_severity(severity), progname, message)
+        caller_object = caller_locations.last
 
-        Systemd::Journal.print(log_level_map[severity], message)
+        # TODO: Allow the syslog facility to be set via the configuration settings.
+        Systemd::Journal.message(
+          :message           => message,
+          :message_id        => message_id,
+          :priority          => log_level_map[severity],
+          :application       => 'manageiq',
+          :syslog_identifier => determine_identifier(message),
+          :syslog_facility   => 'local3',
+          :code_line         => caller_object.lineno,
+          :code_file         => caller_object.absolute_path
```

Comment:
This is awesome :+1:

Reply:
This might not work with LogProxy's `_log` method in ManageIQ, but I think we can solve that in a follow up if it's a problem